### PR TITLE
for fish

### PR DESCRIPTION
**Pull request by hisayan**
I want to use phpenv on fish shell.

### CODE REVIEW

File: libexec/phpenv-init [hunk 1]
```diff
@@ -56,6 +56,9 @@ if [ -z "$print" ]; then
   ksh )
     profile='~/.profile'
     ;;
+  fish )
+    profile='~/.profile'
```
Comment (mizunashi-mana) [inline libexec/phpenv-init]: default startup config path of fish-shell is `~/.config/fish/config.fish`

## See also

* fish document: http://fishshell.com/docs/current/tutorial.html#tut_startup
* rbenv code: https://github.com/rbenv/rbenv/blob/master/libexec/rbenv-init#L61

File: libexec/phpenv-init [hunk 2]
```diff
@@ -73,10 +76,17 @@ fi
 
 mkdir -p "${PHPENV_ROOT}/"{shims,versions}
 
-echo 'export PATH="'${PHPENV_ROOT}'/shims:${PATH}"'
+case "$shell" in
+fish )
+  echo 'set -gx PATH '${NDENV_ROOT}'/shims $PATH'
```
Comment (mizunashi-mana) [inline libexec/phpenv-init]: NDENV -> PHPENV

File: completions/phpenv.fish [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  if [ (count $cmd) -gt 1 ]
+    if [ $argv[1] = $cmd[2] ]
+      return 0
+    end
+  end
+  return 1
+end
+
+complete -f -c phpenv -n '__fish_phpenv_needs_command' -a '(phpenv commands)'
+for cmd in (anyenv commands)
```
Comment (mizunashi-mana) [inline completions/phpenv.fish]: Why do you need anyenv here?

**Review by mizunashi-mana — CHANGES_REQUESTED** [review]

File: libexec/phpenv-init (hunk 1 above)
Comment (hisayan) [reply libexec/phpenv-init]: sorry my mistake. thanks

**Comment by mizunashi-mana** [comment]
Please, add this change: https://github.com/rbenv/rbenv/blob/755c820724162f22d01dd9be5ebf724cb9209c9a/libexec/rbenv-init#L73

File: libexec/phpenv-init [hunk 4]
```diff
@@ -73,10 +83,17 @@ fi
 
 mkdir -p "${PHPENV_ROOT}/"{shims,versions}
 
-echo 'export PATH="'${PHPENV_ROOT}'/shims:${PATH}"'
+case "$shell" in
+fish )
+  echo 'set -gx PATH '${PHPENV_ROOT}'/shims $PATH'
```
Comment (mizunashi-mana) [inline libexec/phpenv-init]: Please, syncing quoted style to rbenv: https://github.com/rbenv/rbenv/blob/755c820724162f22d01dd9be5ebf724cb9209c9a/libexec/rbenv-init#L89

**Review by mizunashi-mana — CHANGES_REQUESTED** [review]

**Comment by mizunashi-mana** [comment]
Please, support phpenv function for fish, too.

## See also

* https://github.com/rbenv/rbenv/blob/755c820724162f22d01dd9be5ebf724cb9209c9a/libexec/rbenv-init#L111

File: libexec/phpenv-init (hunk 4 above)
Comment (mizunashi-mana) [reply libexec/phpenv-init]: @hisayan any update?

**Review by mizunashi-mana — APPROVED** [review]
@hisayan Thanks for changes

LGTM 👍

**Comment by hisayan** [comment]
@mizunashi-mana  Thank you for your kindness.

**Comment by madumlao** [comment]
Was long overdue, sorry bout that!

**Pull request merged**